recipes/docker/steps/00-install.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+# Add Docker's official GPG key:
+
+apt-get install ca-certificates curl gnupg
```

Comment:
Prevent install unexpected additional items, and avoid prompts:

```suggestion
apt-get install -y --no-install-recommends ca-certificates curl gnupg
```